### DISCUSSION TITLE
Error "Forbidden"

### DISCUSSION THREAD

**Discussion by Tamara2211A** [mozilla/hubs#4770] [General Questions & Discussion]
Estimados, consulto. Desde mi universidad estamos intentando entrar varios desde la misma red a una sala y al acceder muestra el error "Forbidden". Luego de presionar F5 muchas veces, logran acceder. Pero alguien puede decirme por qué puede pasar esto?
Me podrán ayudar? 
Gracias!

**Comment by pattersonbl2** [comment]
¿Sigues teniendo problemas con esto?
